tests/test_log_message.py

```
@@ -0,0 +1,78 @@
+
+
+@pytest.mark.asyncio
+async def test_save_attachments_and_stickers() -> None:
+    author = DummyUser(3, "tester")
+    channel = DummyTextChannel(30, "general")
+    message = DummyMessage(author, channel)
+    message.attachments = [DummyAttachment("a.png"), DummyAttachment("b.png")]
+    message.stickers = [DummyAttachment("s.png")]
+    logger = MessageLogger(message=message)
+
+    attachments = await logger._save_attachments()  # noqa: SLF001
```

Pylint (3.3.7)

[warning] Access to a protected member _save_attachments of a client class

(W0212)